addons/hr_holidays/static/src/owl/components/thread/thread.js

```diff
@@ -0,0 +1,51 @@
+odoo.define('hr_holidays.component.Thread', function (require) {
+'use strict';
+
+var Thread = require('mail.component.Thread');
+
+var core = require('web.core');
+const utils = require('web.utils');
+
+var _t = core._t;
+
+utils.patch(Thread, 'hr_holidays_thread', {
+    /**
+     * @override
+     */
+    _useStore: function (state, props) {
```

Comment:
make sections for methods, i.e. `// Public`, `// Private`, etc.

Reply:
ok